build-presets/MakePresetTree.py

```diff
@@ -1,86 +1,131 @@
+    'CdTe': ('fixed tilt, utility scale', 'single-axis tracked, utility scale', 'roof-mounted, commercial scale', 'fixed tilt, commercial scale')
 }
 
 
-# %% Preset values for module parameters: costs are in USD per square meter, efficiency reported as a percentage
+# Preset values for module parameters: costs are in USD per square meter, efficiency reported as a percentage
 module_details = {
-    'cost_front_layer': 4.06,
-    'cost_cell': {'mono-Si': 34.4, 'multi-Si': 31.45, 'CdTe': 30},
-    'cost_back_layer': {'glass-polymer backsheet': 2.32, 'glass-glass': 3},
-    'cost_noncell': 18.,
-    'efficiency': {'mono-Si': 19.0, 'multi-Si': 17.0, 'CdTe': 16.0},
+    'cost_front_layer': 3.5,
+    'cost_cell': {'mono-Si': 22.2, 'multi-Si': 19.4, 'CdTe': 21.3},
+    'cost_back_layer': {'glass-polymer backsheet': 2.4, 'glass-glass': 3},
+    'cost_noncell': 13.6,
+    'efficiency': {'mono-Si': 19.5, 'multi-Si': 17.5, 'CdTe': 18.0},
 }
 
 
-# %% Preset values for operation & maintenance costs, reported in USD/kW(DC) per year
+# Preset values for operation & maintenance costs, reported in USD/kW(DC) per year
 cost_om = {
-    'fixed tilt, utility scale': 15.4,
-    'single-axis tracked, utility scale': 18.50,
-    'roof-mounted, residential scale': 21.00
+    'fixed tilt, utility scale': 16.32,
+    'single-axis tracked, utility scale': 17.46,
+    'roof-mounted, residential scale': 28.94,
+    'roof-mounted, commercial scale': 18.55,
+    'fixed tilt, commercial scale': 18.71
 }
 
+inverter_loading_ratio = [1.1, 1.3, 1.4]
 
-# %%  Read energy yield and degradation rate data, remove weather file types from Location names
-df_yield = pd.read_excel('Energy-yield_deg-rates.xlsx')
-df_yield['Location'] = [label.replace(' (TMY2)', '').replace(' (TMY3)', '') for label in df_yield['Location']]
+# creating PySAM model with default info from json file (that doesn't have location info)
+json_file = open("pvwatts_inputs.json")
+dict = json.load(json_file)
 
+# using the pvwatts model
+pv_dat = pssc.dict_to_ssc_table(dict, "pvwattsv7")
+json_file.close()
+json_model = pvwatts.wrap(pv_dat)
 
-# %% Turn first four columns into keys, and create dictionaries for each energy yield and degradation rate by location
-key_cols = list(df_yield.columns[:4])
-value_cols = df_yield.columns[4:]
-seq_keys = df_yield.groupby(key_cols).apply(lambda sub_df: sub_df[value_cols].values.tolist()).to_dict()
+# convert weather files into format that can be used by PySAM
+weather_folder = "weather_files"
+weather_files = glob.glob(weather_folder + "/*.csv")
+weather_data = [tools.SAM_CSV_to_solar_data(f) for f in weather_files]
 
-system_yield = {k: v[0][1] for k, v in seq_keys.items()}
-degradation_rates = {k: v[0][2] for k, v in seq_keys.items()}
+# tilt angles reported in degrees, needed for running PySAM
+tilt = {'fixed tilt, utility scale': 33, 'single-axis tracked, utility scale': 33,
```

Comment:
single-axis tracked should have tilt=0
```suggestion
tilt = {'fixed tilt, utility scale': 33, 'single-axis tracked, utility scale': 0,
```